control-plane-operator/main.go

```diff
@@ -388,7 +395,43 @@ func NewStartCommand() *cobra.Command {
 			os.Exit(1)
 		}
 
-		if err := (&hostedcontrolplane.HostedControlPlaneReconciler{
+		kubeClient, err := kubernetes.NewForConfig(restConfig)
+		if err != nil {
+			setupLog.Error(err, "could not create k8s client")
+			os.Exit(1)
+		}
+		kubeInformersForNamespaces := v1helpers.NewKubeInformersForNamespaces(kubeClient, namespace)
+		operatorClient, _, err := genericoperatorclient.NewStaticPodOperatorClient(restConfig, operatorv1.GroupVersion.WithResource("kubeapiservers")) // TODO: where do we report status?
```

Comment:
We will remove this when https://github.com/openshift/library-go/pull/1593 merges.